src/ucp/core/ucp_context.h

```diff
@@ -35,12 +35,14 @@ enum {
 
 
 struct ucp_config {
-    /** Array of device names to use */
-    UCS_CONFIG_STRING_ARRAY_FIELD(names)   devices;
+    /** Array of network device names to use */
+    str_names_array_t net_devices;
```

Comment:
what would that save? i mean, i would still always have to pass the device_type for the index in masks..